cmd/root/root.go

```diff
@@ -43,6 +44,7 @@ func NewRootCommand() (*cobra.Command, error) {
 	cmd.AddCommand(generate.NewGenerateCommand())
 	cmd.AddCommand(insights.NewInsightsCommand())
 	cmd.AddCommand(version.NewVersionCommand())
+	cmd.AddCommand(docs.NewDocsCommand())
```

Comment:
Mostly spit balling here: let's mark the docs command as hidden. It's really only a utility command for us to generate markdown docs to then drop in this repo.

And on that note, let's add the docs into this repository to enable @BekahHW to pick those up automatically into our docs site if we want:

```
❯ tree .
.
├── CHANGELOG.md
├── CODEOWNERS
├── Dockerfile
├── LICENSE
├── README.md
├── api
├── build
├── cmd
├── docs
│   ├── pizza.md
│   ├── pizza_completion.md
│   ├── pizza_completion_bash.md
│   ├── pizza_completion_fish.md
│   ├── pizza_completion_powershell.md
│   ├── pizza_completion_zsh.md
│   ├── pizza_docs.md
│   ├── pizza_generate.md
│   ├── pizza_generate_codeowners.md
│   ├── pizza_insights.md
│   ├── pizza_insights_contributors.md
│   ├── pizza_insights_repositories.md
│   ├── pizza_insights_user-contributions.md
│   ├── pizza_login.md
│   └── pizza_version.md
├── go.mod
├── go.sum
├── install.sh
├── justfile
├── main.go
├── npm
└── pkg

26 directories, 84 files
```